basic/src/test/java/org/jboss/hal/testsuite/test/configuration/container/IIOPTestCase.java

```diff
@@ -177,10 +179,13 @@ public void expectErrorForInvalidHighWaterMarkValue() throws Exception {
 
     @Test
     public void highWaterMarkValueTest() throws Exception {
+        String highWaterMark = "high-water-mark";
```

Comment:
I prefer to have the ``highWaterMark`` and ``securityDomain`` as a constant to keep consistent with the rest of the test suite